### ISSUE TITLE
CentOS guide intro incompatible gitlab and gitlab-shell combination

### ISSUE THREAD

**Issue by xilin** [gitlabhq/gitlab-recipes#260]
According to this issue https://gitlab.com/gitlab-org/gitlab-ce/issues/700, 7.4 is incompatible with 2.1.0

**Comment by axilleas** [comment]
Yeah, the guide needs updating :/ You can submit a PR if you like otherwise I'll do it whenever I find some free time.

**Comment by xilin** [comment]
@axilleas I've created a PR.
